provision/tests/run-provisioners.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+. "/srv/provision/tests/provisioners.sh"
+
+pre_hook
+main
+dashboard
+utility_sources
+provision_utilities
+sites
```

Comment:
Lets make sure all of these are prefixed with `provision_`
```suggestion
provision_main
provision_dashboard
provision_utility_sources
provision_utilities
provision_sites
```